package.json

```diff
@@ -12,15 +12,15 @@
     "index.*"
   ],
   "dependencies": {
-    "eslint-visitor-keys": "^1.1.0"
+    "eslint-visitor-keys": "^2.0.0"
```

Comment:
I think this change wouldn't make a difference if `eslint-utils` is used with ESLint >= 4.14.0, since it will get visitor keys from the `sourceCode` object rather than from this package. Other changes seem to be either caused by this one, or internal.

Reply:
I've added the ESLint 5 peerDependency, because that's the minimum version in the tests.